docs/DeveloperGuide.md

```diff
@@ -619,34 +621,33 @@ _{Explain here how the data archiving feature will be implemented}_
-| `* * *`  | user     | find a candidate in the system                          | access details of the candidate without having to go through the entire list. |
-| `* * *`  | user     | list all candidates in the system                       | monitor the application pool.                                                 |
-| `* *`    | user     | sort candidates                                         | reorder candidates based on a specific attribute field for easier viewing.    |
-| `* * *`  | user     | view scheduled interviews within a specific time period | keep track of the upcoming interview schedule.                                |
-| `* * *`  | user     | schedule TA candidates for interviews                   | keep track of the interview schedule.                                         |
-| `* * *`  | new user | view all available commands                             | get familiarised with the system.                                             |
+| Priority | As a …​  | I want to …​                                            | So that I can…​                                                                    |
+|----------|----------|---------------------------------------------------------|------------------------------------------------------------------------------------|
+| `* * *`  | user     | add a candidate to the system                           | keep track of their application details for consideration.                         |
+| `* * *`  | user     | edit a candidate in the system                          | correct any wrong inputs when I added this candidate.                              |
+| `* * *`  | user     | delete a candidate from the system                      | remove entries that I no longer need.                                              |
+| `* * *`  | user     | find a candidate in the system                          | access details of the candidate without having to go through the entire list.      |
+| `* * *`  | user     | list all candidates in the system                       | monitor the application pool.                                                      |
+| `* *`    | user     | sort candidates in the system                           | view the candidates in a more organised manner based on a certain attribute field. |
+| `* * *`  | user     | view scheduled interviews within a specific time period | keep track of the upcoming interview schedule.                                     |
+| `* * *`  | user     | schedule TA candidates for interviews                   | keep track of the interview schedule.                                              |
+| `* * *`  | user     | update the remark for a candidate in the system         | keep a note of important details relating to the candidate.                        |
+| `* * *`  | new user | view all available commands                             | get familiarised with the system.                                                  |
```

Comment:
I'll work on the DG later. Will check if we're missing any user stories and then add it in 👍🏻

Reply:
cool, thank you! not sure if the rest are working on any parts here, maybe can do a quick check w them. I only added for the few commands I was more involved in for now 👍